thirdparty/elementor.cls.php

```diff
@@ -46,4 +46,13 @@ public static function disable_litespeed_esi()
 	{
 		define('LITESPEED_ESI_OFF', true);
 	}
+
+	public static function detect()
+	{
```

Comment:
@timotei-litespeed This should only run when elementor is enabled

Reply:
Added